src/app/services/sms/sms.service.ts

```diff
@@ -186,12 +186,15 @@ const sendSms = (
     smsType,
   }
 
+  const statusCallbackRoute = '/api/v3/notifications/twilio'
+
   return ResultAsync.fromPromise(
     client.messages.create({
       to: recipient,
       body: message,
       from: msgSrvcSid,
       forceDelivery: true,
+      statusCallback: config.app.appUrl + statusCallbackRoute,
```

Comment:
to prevent the URL from desyncing, should we export a constant from the router itself so we have a higher chance of remembering to update it when we update the routes

e.g

```ts
// src/app/routes/api/v3/notifications/notifications.routes.ts
export const ABSOLUTE_TWILIO_NOTIF_ROUTE = '/api/v3/notifications/twilio'

// src/app/services/sms/sms.service.ts
statusCallback: `${config.app.appUrl}${ABSOLUTE_TWILIO_NOTIF_ROUTE}`
```

Reply:
I tried adding this const into `notifications.routes` and imported it into `sms.service` but I get a `Route.post() requires callback functions but got a "object Undefined" error from `routes/.../notifications/bounce.routes`,  which should not be the case since POST '/email` under `bounce.routes` takes in `handleSns`

Seems like this error is due to a circular dependency being created. I traced the dependencies, and there seems to be a dependency in this order `notifications.routes` -> `bounce.routes` -> `bounce.controller` -> `bounce.service` -> `SmsFactory` -> `sms.service` -> `notification.routes`.
Also, managed to console log and find out that `handleSns` becomes `undefined` once I import`ABSOLUTE_TWILIO_NOTIF_ROUTE` into `sms.service

Should we move the const `ABSOLUTE_TWILIO_NOTIF_ROUTE` elsewhere or leave it in `sms.service`

Reply:
i guess we can leave it. Just add a comment in sms routes in the handler to tell future changes to sync to the given file